File: src/main/java/it/frob/sleighidea/psi/SleighElementType.java
// SPDX-License-Identifier: Apache-2.0

package it.frob.sleighidea.psi;

import com.intellij.psi.tree.IElementType;
import it.frob.sleighidea.SleighLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class SleighElementType extends IElementType {
    public SleighElementType(@NotNull @NonNls String debugName) {
        super(debugName, SleighLanguage.INSTANCE);
    }
}

File: src/main/kotlin/it/frob/sleighidea/psi/SleighElementType.kt
// SPDX-License-Identifier: Apache-2.0

package it.frob.sleighidea.psi

import com.intellij.psi.tree.IElementType
import it.frob.sleighidea.SleighLanguage
import org.jetbrains.annotations.NonNls

class SleighElementType(debugName: @NonNls String) : IElementType(debugName, SleighLanguage.INSTANCE)